xmake/modules/package/manager/clib/install_package.lua

```diff
@@ -0,0 +1,80 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- 
+-- Copyright (C) 2015 - 2019, TBOOX Open Source Group.
+--
+-- @author      Adel Vilkov (aka RaZeR-RBI)
+-- @file        install_package.lua
+--
+
+-- imports
+import("core.base.option")
+import("core.project.config")
+import("lib.detect.find_tool")
+-- install package
+-- @param name  the package name, e.g. clib::clibs/bytes@0.4.0
+-- @param opt   the options, .e.g { verbose = true, out_dir = "clib",
+--                                  save = false, save_dev = false }
+--
+-- @return      true or false
+--
+function main(name, opt)
+    -- find clib
+    local clib = find_tool("clib")
+    if not clib then
```

Comment:
Please add `find_clib.lua` to improve to find results. refer to https://github.com/xmake-io/xmake/blob/master/xmake/modules/detect/tools/find_conan.lua